### ISSUE TITLE
Accessibility - Incorrect Link Text Format

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#9]
To be accessible and understandable to s screen reader user (VoiceOver) a links text should be word not the URL. Listening to URL text instead of the proper words is confusing and not easily understood.

**Poor URL Text**:

[https://github.com/USDepartmentofLabor/Ilab-do-good](https://github.com/USDepartmentofLabor/Ilab-do-good)

**Good URL Text**:

[United State Department of Labor Github Repository for ILAB ComplyChain](https://github.com/USDepartmentofLabor/Ilab-do-good)

**NOTE** - My testing is being done on our iPad Air 2 (iOS 10.3.3) because our iPod Touch 5th Generation (iOS 9.3.5) opens the application but displays a blank screen.

**Comment by adriennerlong** [comment]
We have flagged this a low priority issue. We will ask developers to address before 9/11, if possible.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Screen reader users are confused hearing a jumble of text and have no idea where the link is going so most likely won't go there to read what you are wanting them to read.

These should be fixed to make the application useful to all users.

**Comment by jfc3-dol** [comment]
This issue is still outstanding at least for the different Steps Topic screens I checked.
